python/tests/datasets/test_weather.py

```diff
@@ -0,0 +1,120 @@
+from datetime import date, datetime, timezone
+
+import pytest
+
+from whylogs.datasets import Weather
+from whylogs.datasets.configs import WeatherConfig
+
+dataset_config = WeatherConfig()
+
+
+class TestWeatherDataset(object):
+    @pytest.fixture
+    def dataset(self):
+        return Weather(version="in_domain")
```

Comment:
what happens if you don't specify the dataset version?

Reply:
default version is "in_domain". The version here is actually not needed.